### PR TITLE
371 ammended Place code K3, age 16

### LINKED ISSUE #699: Error code 371 [Amended 2024 v1.0]
#### Error code
371
#### Description to display
Child in ~~semi-independent living accommodation not subject to children’s homes regulations~~ **supported accommodation** should be at least 16.
#### DfE Long-form description of error
Where the placement code is ~~H5~~ **K3, then** the date at the start of the placement should be on or after the child’s 16th birthday.
#### DfE suggested coding
Where `<PL>` = ~~‘H5’~~ **K3** then `<DOB>` should be >= 16 years prior to `<DECOM>`

For further info, refer to the DfE validation document [here](https://www.gov.uk/government/publications/children-looked-after-return-2024-to-2025-technical-specifications)

### PR DESCRIPTION
**Pull request by WillLP-code**
closes #699 
Updates 371 for 23/24 by changing code to K3 from H5 and updating age from 14 to 16

### CODE REVIEW

File: lac_validator/rules/lac2023_24/rule_371.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            "CHILD": ["111", "222", "333", "444", "555", "666"],
+            "DECOM": [
+                "01/01/2022",
+                "01/02/2022",
+                "02/03/2022",
+                "15/01/1982",
+                "15/03/2016",
+                "14/03/2016",
+            ],
+            "PLACE": ["K3", "P2", "P2", "K3", "K3", "K3"],  # 14bd    #day before 14bd
```
Comment (SLornieCYC) [inline lac_validator/rules/lac2023_24/rule_371.py]: Comment needs updating to 16bd.

**Review by SLornieCYC — CHANGES_REQUESTED** [review]
This one is fine, apart from the change comes in from the **2024/2025** return (not 2023/2024) so the rule file needs moving forward a year in the folder structure.

File: lac_validator/rules/lac2023_24/rule_371.py (hunk 1 above)
Comment (WillLP-code) [reply lac_validator/rules/lac2023_24/rule_371.py]: As with 222, I'll do this when 218 is merged in as it has the folder/init structure for 24/25 :)

**Review by SLornieCYC — APPROVED** [review]
Think this one is good now!

**Pull request merged**